src/storage_raft_member/raft_member.rs

```diff
@@ -216,9 +216,10 @@ impl RaftNetwork<WormFsTypeConfig> for RaftMember {
     ) -> Result<AppendEntriesResponse<NodeId>, RPCError<NodeId, WormFsNode, RaftError<NodeId>>>
     {
         // Log the AppendEntries request details
-        debug!(
-            "[RaftMember] Sending AppendEntries to {:?}: term={}, prev_log={:?}, entries={}, leader_commit={:?}",
+        info!(
```

Comment:
Some of these log lines will be way to verbose in prod. we will need to do a clean up in the future to both map to the right levels but also make some debug compile time conditional.s